Custom build fails for component charts

wanted to make a custom build which includes component button and charts.
ran the following command.

npm run build -- --components=button,charts

get the below error:

`temp/index.js → dist/js/sohoxi.js...
[!] Error: 'Charts' is not exported by src\components\charts\charts.js, imported by temp\components.js
https://rollupjs.org/guide/en/#error-name-is-not-exported-by-module
temp\components.js (4:9)
2: export { Button } from '../src/components/button/button';
3: export { ButtonSet } from '../src/components/button/button.set';
4: export { Charts } from '../src/components/charts/charts';
            ^
5:
6: // Mid ====/
Error: 'Charts' is not exported by src\components\charts\charts.js, imported by temp\components.js
    at error (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:151:30)
    at Module.error (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:10041:16)
    at Module.getVariableForExportName (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:10204:29)
    at Module.includeAllExports (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:10279:35)
    at Object.includeAllExports (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:10373:43)
    at NamespaceVariable.include (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:9627:22)
    at Module.includeVariable (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:10697:22)
    at Module.includeAllExports (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:10274:26)
    at Graph.includeStatements (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:20394:36)
    at Graph.build (C:\projects\enterprise\node_modules\rollup\dist\shared\rollup.js:20309:14)

ⓧ  Error: "npx rollup -c --components=button,charts" process exited with error code (1)
Args:
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ids-enterprise@4.54.0-dev build: `npm run clean:dist && node ./scripts/build "--components=button,charts"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ids-enterprise@4.54.0-dev build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\nchandrasoma\AppData\Roaming\npm-cache\_logs\2021-07-27T12_44_22_058Z-debug.log
`

--- a/docs/CHANGELOG.md
+++ b/docs/CHANGELOG.md
@@ -3,6 +3,7 @@
 ## v4.55.0 Features
 
 - `[ApplicationMenu]` Added the ability to resize the app menu. ([#5193](https://github.com/infor-design/enterprise/issues/5193))
+- `[Custom Builds]` Fixed a bug where importing the base Charts API directly would cause an error. ([#5463](https://github.com/infor-design/enterprise/issues/5463))
 - `[Datagrid]` Adds the ability to have a selection radio buttons on Datagrid. ([#5384](https://github.com/infor-design/enterprise/issues/5384))
 - `[Datagrid]` Added a `verticalScrollToEnd` property when you reached the end of the datagrid list. ([#5435](https://github.com/infor-design/enterprise/issues/5435))
 - `[Datagrid]` Added seperate mask options for filter row. ([#5519](https://github.com/infor-design/enterprise/issues/5519))

--- a/scripts/build.js
+++ b/scripts/build.js
@@ -238,6 +238,7 @@ const changedExportNames = {
 };
 
 const lowercaseConstructorNames = {
+  charts: 'charts',
   masks: 'masks',
   longpress: 'longPress',
   renderloop: 'renderLoop'

--- a/src/components/charts/_charts-new.scss
+++ b/src/components/charts/_charts-new.scss
@@ -1,6 +1,9 @@
 // Uplift Chart - Svg and Css
 //================================================== //
 
+@import '../popupmenu/popupmenu-new';
+@import '../tooltip/tooltip-new';
+
 .chart-container {
   .chart-legend.is-bottom {
     padding-bottom: 0;

--- a/src/components/charts/_charts.scss
+++ b/src/components/charts/_charts.scss
@@ -1,6 +1,9 @@
 // Chart - Svg and Css
 //==================================================//
 
+@import '../popupmenu/popupmenu';
+@import '../tooltip/tooltip';
+
 // Alternate Text message
 .chart-message {
   background-color: $ids-color-palette-ruby-10;
@@ -300,7 +303,7 @@
 // Legend Popup Styling
 .chart-popup-menu {
   display: flex;
-  
+
   .chart-popup-menu-color {
     height: 15px;
     width: 15px;

--- a/src/components/charts/charts.js
+++ b/src/components/charts/charts.js
@@ -5,6 +5,7 @@ import { DOM } from '../../utils/dom';
 import { theme } from '../theme/theme';
 import { Locale } from '../locale/locale';
 
+import '../popover/popover.jquery';
 import '../popupmenu/popupmenu.jquery';
 
 const charts = {};

